src/components/common/Stepper.vue

```diff
@@ -10,12 +10,12 @@
             :id=step.id
             class="step__btn"
             tabindex="-1"
-            :disabled=step.disabled
+            :disabled=isBusySaving
             :ripple="false"
             :class="{ 'selected-btn': isCurrentStep(step) }">
             <v-icon class="step__icon" :class="{ 'selected-icon': isCurrentStep(step) }">{{ step.icon }}</v-icon>
           </v-btn>
-          <v-icon class="step__btn2" size="30" color="green darken-1" v-show=step.valid>
+          <v-icon class="step__btn2" size="30" color="green darken-1" v-show=isValid(step.to)>
             mdi-check-circle
           </v-icon>
         </div>
```

Comment:
While you're here, you might want to implement "isInvalid" (with a X circle icon).

Reply:
Was my thoughts as well! 
Although we currently only have two states `true/false` so would need to apply this validation at some point prompted by the user to follow our current pattern, surely not display an X all the time eh?

Reply:
I believe there are 3 states:
- initial (no validation applied)
- valid when complete/valid
- invalid when incomplete and validation has been applied (via route param I think)

Fortunately, JS booleans have at least 3 states. (joke but also true)

Reply:
I think due to the time being already a day over without including Unit Test Updates, will need to push back on adding this. 
Mainly due to no UX input here, we would only be guessing what to show and when, right?

Reply:
I saw the "invalid X" in a comp somewhere.

Just add a todo checkbox for this item to another ticket then.

Reply:
Will do. 

Perhaps this was in the Coops designs we just looked at? As thats the only designs in progress with regards to Create UI afaik.

Reply:
Haha, I think you're right on both counts!